### ISSUE TITLE
Errors on linked page

### ISSUE THREAD

**Issue by tonyhallett** [mdn/content#29255]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_Colors_and_Luminance

### What specific section or headline is this issue about?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_Colors_and_Luminance#luminance

### What information was incorrect, unhelpful, or incomplete?

The linked page https://alvaromontoro.com/blog/67854/building-your-own-color-contrast-checker provides in its theory section 

> [Relative luminance is] the relative brightness of any point in a colorspace, normalized to 0 for darkest black and 1 for lightest white.

> the Contrast Ratio Formula.

> CR = (L1 + 0.05) / (L2 + 0.05), where
L1 is the relative luminance of the lighter of the colors, and
L2 is the relative luminance of the darker of the colors.

This is correct but then in its Calculate the contrast ratio section states

> As we explained in the theory part, the luminance is a value between 0 and 1. That means that the color with higher luminance (the darkest one) will be used as the dividend in the formula:
// calculate the color contrast ratio
const ratio = color1luminance > color2luminance 
    ? ((color2luminance + 0.05) / (color1luminance + 0.05))
    : ((color1luminance + 0.05) / (color2luminance + 0.05));

The color with the highest luminance is not the darkest one.  L1, which forms the dividend, is the lighter of the two colors.  
It then puts the color with the higher luminance in the divisor.

Given how it has done the calculation  

```
// show results depending on WCAG requirements
const result = `
                AA-level large text: ${ratio < 1/3 ? 'PASS' : 'FAIL' }<br>
                AA-level small text: ${ratio < 1/4.5 ? 'PASS' : 'FAIL' }<br>
                AAA-level large text: ${ratio < 1/4.5 ? 'PASS' : 'FAIL' }<br>
                AAA-level small text: ${ratio < 1/7 ? 'PASS' : 'FAIL' }
               `;
document.querySelector("#result").innerHTML = result;

```

Instead of ratio >=

https://www.w3.org/TR/WCAG21/#contrast-minimum

> The visual presentation of [text](https://www.w3.org/TR/WCAG21/#dfn-text) and [images of text](https://www.w3.org/TR/WCAG21/#dfn-images-of-text) has a [contrast ratio](https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio) of at least 4.5:1, 

### What did you expect to see?

Correct content

### Do you have any supporting links, references, or citations?

https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio

### Do you have anything more you want to share?

It is also using the old formula for relative luminance.

```
function luminance(r, g, b) {
    var a = [r, g, b].map(function (v) {
        v /= 255;
        return v <= 0.03928
            ? v / 12.92
            : Math.pow( (v + 0.055) / 1.055, 2.4 );
    });
    return a[0] * 0.2126 + a[1] * 0.7152 + a[2] * 0.0722;
}
```

https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast links to 
https://www.w3.org/TR/WCAG21/#dfn-relative-luminance

> Before May 2021 the value of 0.04045 in the definition was different (0.03928). It was taken from an older version of the specification and has been updated. It has no practical effect on the calculations in the context of these guidelines.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility/understanding_colors_and_luminance`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_Colors_and_Luminance
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/understanding_colors_and_luminance/index.md
* Last commit: https://github.com/mdn/content/commit/07f0cf4375aaa02e1071d8bd0e8518db7609b7a9
* Document last modified: 2023-07-17T23:05:13.000Z

</details>
